source/docs/software/commandbased/organizing-command-based.rst

```diff
@@ -0,0 +1,313 @@
+        return new StartEndCommand(() -> this.set(percent), () -> this.set(0), this);
+    }
+
+  .. code-tab:: c++
+
+    // TODO
+
+For instance, this code creates a command group that runs the intake forwards for two seconds, waits for two seconds, and then runs the intake backwards for five seconds.
+
+.. tabs::
+
+  .. code-tab:: java
+
+    Command intakeRunSequence = intake.commandRun(1).withTimeout(2)
+        .andThen(new WaitCommand(2))
+        .andThen(intake.commandRun(-1).withTimeout(5));
+
+  .. code-tab:: c++
+
+    // TODO
+
+Static Command Factories
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The factory methods presented above do have some negatives. They clutter up the `Intake` class (and the namespace of `Intake` methods) with command-related methods. With that pattern, it's also possible to break the intended encapsulation of the subsystem (as the warning box alludes to), since code in these factory methods can access `private` members of the `Intake` class. To avoid these pitfalls, it's possible to define intake-related methods as static members of a separate `IntakeCommands` class:
+
+.. tabs::
+
+  .. code-tab:: java
+
+    public class IntakeCommands {
+        public static Command run(Intake intake, double percent) {
+            return new StartEndCommand(() -> intake.set(percent), () -> intake.set(0), intake);
+        }
+    }
+
+  .. code-tab:: c++
+
+    // TODO
+
+However, these benefits come at the cost of somewhat more cumbersome usage code. This is what the intake-then-outtake sequence looks like using static factory methods.
+Notice how ``intake.commandRun(1)`` has been replaced with ``IntakeCommands.run(intake, 1)``, but the code is otherwise identical:
+
+.. tabs::
+
+  .. code-tab:: java
+
+    Command intakeRunSequence = IntakeCommands.run(intake, 1).withTimeout(2)
+        .andThen(new WaitCommand(2))
+        .andThen(IntakeCommands.run(intake, 1).withTimeout(5));
+
+  .. code-tab:: c++
+
+    // TODO
+
+The trade-offs between static and instance factory methods for single subsystems make the choice between them entirely a matter of personal opinion, and shouldn't be the subject of too much concern. Some programming teams might prefer static methods to maintain consistency with multi-subsystem command groups, in order to define commands consistently across the entire codebase. However, other teams might want to take advantage of the more fluid syntax offered by instance methods. For the command group discussion below, this article uses instance method calls.
+
+.. note:: Much like command groups, commands that require multiple subsystems should not be defined in instance methods of subsystems, since they are not conceptually related to a single subsystem. These commands should be either defined inline at their usage location, constructed using a static factory method, or made a subclass of ``CommandBase`` (depending on complexity of the command and programmer preference).
```

Comment:
```suggestion
.. note:: Commands that require multiple subsystems should not be defined in instance methods of subsystems, since they are not conceptually related to a single subsystem. These commands should be either defined inline at their usage location (if used in only one location) or constructed using one of the other options detailed in this article, depending on the complexity of the command and programmer preference).
```
No reason to split commands vs groups.

This should be moved to the section about instance factory methods.